src/SSHLibrary/library.py

```diff
@@ -832,9 +833,16 @@ def login_with_public_key(self, username, keyfile, password='',
         | Login With Public Key | johndoe          | /home/johndoe/.ssh/id_dsa | keyringpasswd |
 
         Argument `delay` was added in SSHLibrary 2.0.
+
+        To provide SSH agent support, `allow_agent` and `look_for_keys` were introduced in
+        SSHLibrary 3.0.0.
+        `allow_agent` enables the connection to the SSH agent.
+        `look_for_keys` enables the searching for discoverable private key files in `~/.ssh/`.
+        `allow_agent` and `look_for_keys` were not available prior to SSHLibrary 3.0.0 and are
+         ignored by Jython.
         """
         return self._login(self.current.login_with_public_key, username,
-                           keyfile, password, delay)
+                           keyfile, password, allow_agent, look_for_keys, delay)
```

Comment:
It would be good to use `is_truthy()` here.

Reply:
Done.